docs/pages/kmodal.vue

```diff
@@ -11,6 +11,65 @@
         text="modals"
       />.
     </DocsPageSection>
+    <DocsPageSection
+      title="Usage"
+      anchor="#usage"
+    >
+      <DocsSubNav
+        :items="[
+          { text: 'Dialog box with description', href: '#dialog-box' },
+          { text: 'Submit form dialog box', href: '#submit-form' },
+          { text: 'Modal with different sizes', href: '#modal-sizes' },
+          { text: 'Modal with precised sizes', href: '#modal-dynamic-size' },
+        ]"
+      />
+
+      <h3>
+        Dialog box with description
+        <DocsAnchorTarget anchor="#dialog-box" />
+      </h3>
+      <p>To show information, warnings, or notifications.</p>
+      <DocsExample
+        loadExample="KModal/ModalWithDescription.vue"
+        exampleId="dialog-box"
+      />
+
+      <h3>
+        Submit form dialog box
+        <DocsAnchorTarget anchor="#submit-form" />
+      </h3>
+      <p>
+        Modal with checkbox, submit, and cancel buttons <br >
+        Check the checkbox to enable the submit button <br >
+        To discard the changes, click cancel
+      </p>
+      <DocsExample
+        loadExample="KModal/ModalWithSubmit&Cancel.vue"
+        exampleId="submit-form"
+      />
+
+      <h3>
+        Modal with different sizes
```

Comment:
```suggestion
        Modal with predefined sizes
```

To make clearer the distinction between this section and the new precise size section.

Reply:
Resolved PR comments